src/python/pants/engine/internals/synthetic_targets.py

```diff
@@ -152,54 +149,7 @@ def rules(cls) -> Iterator[UnionRule]:
 
 
 class SyntheticAddressMap(AddressMap):
-    def process_declared_targets(self, address_map: AddressMap) -> None:
-        for name, target_adaptor in address_map.name_to_target_adaptor.items():
-            extend_synthetic = target_adaptor.kwargs.pop("_extend_synthetic", False)
-            if name not in self.name_to_target_adaptor:
-                if extend_synthetic:
-                    raise InvalidTargetException(
-                        softwrap(
-                            f"""
-                            The `{target_adaptor.type_alias}` target {name!r} in {address_map.path}
-                            has `_extend_synthetic=True` but there is no synthetic target to extend.
-                            """
-                        )
-                    )
-                continue
-
-            # Pop synthetic target to let the explicit target declared in BUILD file take
-            # precedence.
-            synthetic_target_adaptor = self.name_to_target_adaptor.pop(name)
-
-            if not extend_synthetic:
-                # The explicitly declared target should replace the synthetic one.
-                continue
-
-            # Check target type matches, when marked as extending the synthetic target.
-            if synthetic_target_adaptor.type_alias != target_adaptor.type_alias:
-                raise InvalidTargetException(
-                    softwrap(
-                        f"""
-                        The `{target_adaptor.type_alias}` target {name!r} in {address_map.path} is
-                        of a different type than the synthetic target
-                        `{synthetic_target_adaptor.type_alias}` from {self.path}.
-
-                        When `_extend_synthetic` is true the target types must match, set this to
-                        false if you want to replace the synthetic target with the target from your
-                        BUILD file.
-                        """
-                    )
-                )
-
-            # Preserve synthetic field values not overriden by the declared target from the BUILD.
-            synthetic_target_adaptor.kwargs.update(target_adaptor.kwargs)
-            target_adaptor.kwargs = synthetic_target_adaptor.kwargs
-
-    def apply_defaults(self, defaults: BuildFileDefaults) -> None:
-        for target_adaptor in self.name_to_target_adaptor.values():
-            default_values = defaults.get(target_adaptor.type_alias)
-            if default_values is not None:
-                target_adaptor.kwargs = {**default_values, **target_adaptor.kwargs}
+    ...
```

Comment:
Would `pass` be more idiomatic? `...` seems to be for type stubs from what I have seen.

Reply:
Yeah, probably, although I've seen both I think?

Reply:
Will change to `pass`